### PR TITLE
OCPBUGS#1653: Rapidly changing clusters can lower the maximum practical size of a cluster

### PR DESCRIPTION
**Pull request by xenolinux**
Rapidly changing clusters can lower the maximum practical size of a cluster

Version(s):  4.9+

Issue: https://issues.redhat.com/browse/OCPBUGS-1653

Link to docs preview: https://56518--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/planning-your-environment-according-to-object-maximums.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc [hunk 1]
```diff
@@ -12,6 +12,11 @@ These guidelines are based on the largest possible cluster. For smaller clusters
 
 In most cases, exceeding these numbers results in lower overall performance. It does not necessarily mean that the cluster will fail.
 
+[NOTE]
+====
+Rapidly changing clusters for example, lots of starting and stopping pods can lower the maximum practical size of a cluster.
```
Comment (kowen-rh) [inline scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc:17]: This seems a bit vague to me. I'd suggest something like this:

```adoc
Clusters that experience rapid change, such as those with many starting and stopping pods, can have a lower practical size than documented.
```

I also wonder if this admonition might be better labeled as a warning. What do you think?

**Review by kowen-rh — COMMENTED** [review]
Thank you for adding this note, the information is very valuable! I do think that the wording can be clearer, so I recommend considering something along the lines of my suggestion. :slightly_smiling_face: 

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc (hunk 1 above)
Comment (xenolinux) [reply scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc]: Changed the note to warning - Addressed

**Pull request merged**